debian/control

```diff
@@ -68,16 +61,53 @@ Depends: ${misc:Depends}, net-tools, ifupdown, htop, isc-dhcp-client,
 # add some tools:
   tree, bzip2, wget, telnet, man, psmisc,
 # molior:
-  molior-server, molior-web, aptly (>= 1.3.0.adsy7)
+  molior-server, molior-client-http, molior-web, aptly (>= 1.3.0.adsy7)
 Description: molior ISO installer for testing
  Testing setup for molior
 
-Package: molior-node-devel
+Package: molior-node-amd64
 Architecture: all
 Pre-Depends: molior-os-config
 Depends: ${misc:Depends}, net-tools, ifupdown, htop, isc-dhcp-client,
-  iputils-ping, bash-completion, vim, openssh-server, linux-image-amd64,
-  grub-pc, rsyslog, acpi-support-base, acpid, sudo, locales, apt-transport-https,
+  iputils-ping, bash-completion, vim, openssh-server,
+  linux-image-amd64, grub-pc,
+  rsyslog, acpi-support-base, acpid, sudo, locales,
+  dialog, bsdmainutils, less, gawk, systemd-sysv, bash-completion,
+  libpam-systemd,
+# editors
+  vim-nox,
+# add some tools:
+  tree, bzip2, wget, telnet, man, psmisc,
+# molior:
+  molior-client-http
+Description: molior build node ISO installer for testing
+ Testing setup for molior
+
+Package: molior-node-amd64-efi
+Architecture: all
+Pre-Depends: molior-os-config
+Depends: ${misc:Depends}, net-tools, ifupdown, htop, isc-dhcp-client,
+  iputils-ping, bash-completion, vim, openssh-server,
+  linux-image-amd64, grub-efi-amd64,
```

Comment:
There is `grub-efi` package available so you don't have to specify architecture.

Reply:
indeed, need to try it out :)

Reply:
unfortunately grub-efi exists only for amd64/i386 and brings in ia32 dependencies...